packages/thirdweb/src/extensions/lens/read/getFullProfile.ts

```diff
@@ -0,0 +1,90 @@
+import type { ThirdwebClient } from "../../../client/client.js";
+// import { getDefaultHandle } from "src/exports/extensions/lens.js";
+import type { ThirdwebContract } from "../../../contract/contract.js";
+import { getProfile } from "../__generated__/LensHub/read/getProfile.js";
+import { getDefaultHandle } from "../__generated__/TokenHandleRegistry/read/getDefaultHandle.js";
+import type { LensProfileSchema } from "./type.js";
+
+export type GetFullProfileParams = {
+  profileId: bigint;
+  client: ThirdwebClient;
+
+  // Lens's smart contracts.
+  // Go here for the latest data: https://www.lens.xyz/docs/resources/smart-contracts
+  lensHubContract: ThirdwebContract;
+  lensHandleContract: ThirdwebContract;
+  tokenHandleRegistryContract: ThirdwebContract;
```

Comment:
these should be optional, default to the deployed ones on polygon. otherwise it makes the API a bit hard to use

Reply:
Sg! I was just thinking about that actually